src/components/TableBody.js

```diff
@@ -159,6 +160,7 @@ class TableBody extends React.Component {
                           {...(columns[columnIndex].setCellProps
                             ? columns[columnIndex].setCellProps(column, dataIndex, columnIndex)
                             : {})}
+                          data-testid={`${columnIndex}-${rowIndex}-MuiDataTableBodyCell`}
```

Comment:
Is there a reason to put the indexes in front of the name here? Putting the name first makes parsing into it easier, e.g. `startsWith`.